engines/bastion_katello/app/assets/javascripts/bastion_katello/capsule-content/capsule-content.routes.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * @ngdoc object
+ * @name Bastion.capsule-content.config
+ *
+ * @requires $stateProvider
+ *
+ * @description
+ *   Used for systems level configuration such as setting up the ui state machine.
+ */
+angular.module('Bastion.capsule-content').config(['$stateProvider', function ($stateProvider) {
+
+    // Fake state to prevent redirects
```

Comment:
Without state routing ends up in router's `otherwise` https://github.com/Katello/bastion/blob/master/app/assets/javascripts/bastion/bastion.module.js#L60-L63 The function does some legacy url transformation I guess. In our case it assigns the same url to `location.url` which causes the reload.